Catappult/appcoins-wallet-android
# [APPC-2228] Wallet Dev - Update Dev Environment Links

**What does this PR do?**

   Updated some dev links used by wallet to the new values: myappcoins for local payments in dev; appcexplorer in ropsten network for transaction details

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] build.gradle (app)

**How should this be manually tested?**

  Consult the ticket (section steps to reproduce) to see how to test this: [APPC-2228](https://aptoide.atlassian.net/browse/APPC-2228)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2228](https://aptoide.atlassian.net/browse/APPC-2228)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)
   Answer: No

**Code Review Checklist**
